src/control/common/proto/mocks.go

```diff
@@ -181,10 +180,7 @@ func (m *MockMgmtSvcClient) PoolSetProp(ctx context.Context, req *mgmtpb.PoolSet
 
 // returnACLResult returns the mock ACL results - either an error or an ACLResp
 func (m *MockMgmtSvcClient) returnACLResult() (*mgmtpb.ACLResp, error) {
-	if m.Cfg.ACLRet.Err != nil {
-		return nil, m.Cfg.ACLRet.Err
-	}
-	return &mgmtpb.ACLResp{ACL: m.Cfg.ACLRet.Acl, Status: m.Cfg.ACLRet.Status}, nil
+	return nil, nil
```

Comment:
If all the ACL mocking is removed from this config, should probably remove this helper and clean up the parent methods. At the very least the comment is wrong now.

Reply:
Yeah, this was mostly to make the compiler happy. I think most if not all of this MockMgtSvcClient will get nuked pretty soon. Tom is working on converting the server fanout stuff over to using the control api instead of using the gRPC client directly.